src/wstool/config_yaml.py

```diff
@@ -386,7 +385,8 @@ def generate_config_yaml(config, filename, header):
     with open(os.path.join(config.get_base_path(), filename), 'w+b') as f:
         if header is not None:
             f.write(header.encode('UTF-8'))
-        items = [x.get_legacy_yaml() for x in config.get_source()]
+        items = [x.get_legacy_yaml() for x in
+                 sorted(config.get_source(), key=lambda x:x.get_local_name())]
```

Comment:
I believe I said that rosws must not be affected. Sorting here probably affects rosws, no?